corehq/apps/registration/static/registration/js/new_user.ko.js

```diff
@@ -229,6 +229,8 @@ hqDefine('registration/js/new_user.ko', [
 
         var _getDataForSubmission = function () {
             var password = self.password();
+            var captcha0 = $(containerSelector).find("[name='captcha_0']")[0].value;
+            var captcha1 = $(containerSelector).find("[name='captcha_1']")[0].value;
```

Comment:
oh, that. ajax form submissions!

Reply:
can we give have better names for these, `captcha_input`, `captcha_confirmation`? or these are used by captcha so we can't change them.
Also, this would get tricky that there is no server side validation right? I can edit the form to put a different but matching value in those fields and that will work too.

Reply:
changed the names for variables but input params should be like that only. The validation happens on the server side when form.valid is called. Basically value will be stored in CaptchaStore while generating  and hash will be forwarded to UI. on submit, captcha_0 has the hash and on backend it will be verified in the backend from captchaStore